### ISSUE TITLE
Ensure Skip to Content has sufficient contrast

### ISSUE THREAD

**Issue by paulcpederson** [Esri/calcite-web#995]
Skip to content should have a background color _and_ text color declared to ensure that it is always readable. Right now we set background to white, but it's possible applications consuming Calcite Web could set the text as white, making it unreadable.

**Comment by paulcpederson** [comment]
will be part of 1.2.0
